test/TesterInternal/GeoClusterTests/BasicLogTestGrainTests.cs

```diff
@@ -10,28 +13,39 @@
 
 namespace Tests.GeoClusterTests
 {
-    public class BasicLogTestGrainTests : TestingSiloHost
+    public class BasicLogTestGrainTests : IClassFixture<BasicLogTestGrainTests.Fixture>
     {
+        private readonly Fixture fixture;
+        private Random random;
+
+        public class Fixture : BaseTestClusterFixture
+        {
+            protected override TestCluster CreateTestCluster()
+            {
+                var options = new TestClusterOptions(1);
+
+                options.ClusterConfiguration.AddMemoryStorageProvider("Default");
+                options.ClusterConfiguration.AddMemoryStorageProvider("MemoryStore");
+                options.ClusterConfiguration.AddAzureTableStorageProvider("AzureStore");
 
+                //AdjustConfig = cfg => LogConsistencyProviderConfiguration.ConfigureLogConsistencyProvidersForTesting(TestDefaultConfiguration.DataConnectionString,cfg)
+                options.ClusterConfiguration.AddAzureTableStorageProvider();
+                options.ClusterConfiguration.AddStateStorageBasedLogConsistencyProvider();
+                options.ClusterConfiguration.AddLogStorageBasedLogConsistencyProvider();
+                options.ClusterConfiguration.AddCustomStorageInterfaceBasedLogConsistencyProvider("CustomStorage");
+                options.ClusterConfiguration.AddCustomStorageInterfaceBasedLogConsistencyProvider("CustomStoragePrimaryCluster");
```

Comment:
Good catch on the props.   I will add that for consistency.  perhaps @sebastianburckhardt could validate if its needed.

I will remove the comment as well and push